test/image-server.etags.json

```diff
@@ -68,5 +68,7 @@
 	"http://localhost:13337/Portrait_5.jpg/:/rs=w:320?cache=false": "3340085419",
 	"http://localhost:13337/Portrait_6.jpg/:/rs=w:320?cache=false": "2612685063",
 	"http://localhost:13337/Portrait_7.jpg/:/rs=w:320?cache=false": "783772049",
-	"http://localhost:13337/Portrait_8.jpg/:/rs=w:320?cache=false": "4137304836"
+	"http://localhost:13337/Portrait_8.jpg/:/rs=w:320?cache=false": "4137304836",
+	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,at:100,al:200/fm=f:jpeg?cache=false": "2536073760",
+	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,ay:100,ax:200/fm=f:jpeg?cache=false": "2536073760"
```

Comment:
My point is this test is producing the same result as `http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,a:tl/fm=f:jpeg?cache=false` -- which should not be the case since they are using different anchor points. Am I missing something, or is this broken?